assets/css/_about_page.scss

```diff
@@ -0,0 +1,34 @@
+.m-about-page {
+  background: $color-bg-medium;
+}
+
+.m-about-page__container {
+  background: $color-bg-light;
+  padding: 2rem;
+  max-width: 30rem;
+}
+
+.m-about-page__title {
+  @include font-header;
+  text-align: center;
+}
+
+.m-about-page__section {
+  background-color: $white;
+  border-radius: 8px;
+  box-shadow: $subtle-box-shadow;
+  margin-bottom: 1rem;
+  padding: 0.5rem 1rem;
+}
+
+.m-about-page__header {
+  @include font-emph;
+  font-weight: bold;
+}
+
+.m-about-page__link {
+  @include button-primary;
+  display: block;
+  text-align: center;
+  text-decoration: none;
```

Comment:
`text-decoration:none` might be good to drop in `@mixin button-base` so it covers any future links that look like buttons.

Alternatively if we want to be more aggressive, it could go in `base.scss` and apply to all `<a>` tags. (If you do this, it can be removed from `@mixin font-link`.)